### ISSUE TITLE
Duplicate identifier 'export=' With Vue Typescript

### ISSUE THREAD

**Issue by hanatharesh2712** [antoine92190/vue-advanced-chat#114]
## Describe the bug
I have added this package in Vue TypeScript project and I get the error while building the Vue project using the "`npm run build`" command.

```
node_modules/vue-advanced-chat/types/vue-advanced-chat.common.d.ts
Error: Duplicate identifier 'export='.
```
I was using the version 0.5.3 without any issues. but, after upgrading it to `0.6.5`. I get the above error.

Package versions:

"vue": "^2.6.12",
"typescript": "^3.9.7",
"eslint": "^5.16.0",

**Comment by kanemotos** [comment]
me too.

I found this code.


`node_modules/vue-advanced-chat/types/vue-advanced-chat.common.d.ts`
```
export = global;
export = global; # <= This is Duplicate identifier.
declare var global: any;
```

I have deleted a second line for work around.
But of course, the error comes again when I do `yarn install` (or `npm install`).

**Comment by antoine92190** [comment]
Could you check if the issue is solved when installing the latest git version: `npm i github:antoine92190/vue-advanced-chat`

**Comment by hanatharesh2712** [comment]
@antoine92190 Yes, It resolves the issue and compiles successfully.

**Comment by hanatharesh2712** [comment]
currently, the workaround is the import the component using `require`.

`const ChatWindow: any = require('vue-advanced-chat').default;`

**Comment by kanemotos** [comment]
@antoine92190 
Thank you for your very quick fix!

@hanatharesh2712 
Thank you for great workaround!! I will try this :)

**Comment by antoine92190** [comment]
@kanemotos @hanatharesh2712 Could you confirm the issue is fixed in v0.6.6?

**Comment by hanatharesh2712** [comment]
@antoine92190 I have tried with v0.6.6 and still, facing the same issues with the build.

**Comment by antoine92190** [comment]
Mmm I cannot reproduce the issue when creating a brand new Vue project with typescript.
Could you create a minimal repo reproducing the issue? Otherwise it will be difficult for me to fix it properly.

**Comment by jcosta33** [comment]
I'm facing the same issue. It happened the first time I tried to use the library, I just removed the duplicate and now it works fine but we have automatic builds and deployments through bitbucket that don't use our local instance of node_modules so I'm still stuck with this problem.

**Comment by antoine92190** [comment]
@jcosta33 I can fix this issue ASAP, but I need to be able to reproduce.
Have you tried to reproduce with a brand new vue project?
If yes, could you share this new project with me?

**Comment by hanatharesh2712** [comment]
@antoine92190 I have tried to reproduce this issue with new Vue + TypeScript project and it builds successfully. so, I investigated with my `tsconfig.json` file and found that the `"skipLibCheck": true,` option is present in the newly created project. but, that was missing in my project.

so, To reproduce this in your project, You can remove `"skipLibCheck": true,`  and then build the project.
If you can check why there is two same below lines and resolve this issue then it will be great.
```
export = global;
export = global;
```

**Comment by antoine92190** [comment]
@hanatharesh2712 Thanks, but I still cannot reproduce after removing `"skipLibCheck": true`.
Could you share your `tsconfig.json`? Or even better a zip file of your new Vue + Typescript project.
I'm don't know why there are duplicate line, I already investigated a lot without any success.
But it will be easier to find a solution when I can reproduce.

**Comment by hanatharesh2712** [comment]
@antoine92190 You can build this repo to reproduce the issue: https://github.com/hanatharesh2712/vue-demo-app

**Comment by antoine92190** [comment]
@hanatharesh2712 Thanks I can reproduce now. Will try to fix it shortly

**Comment by antoine92190** [comment]
@hanatharesh2712 Could you confirm the issue is fixed in v0.6.7?

**Comment by hanatharesh2712** [comment]
@antoine92190 Yes, Now it is working. Thank you so much!!
